build/hellobuild.md

```diff
@@ -133,17 +133,22 @@ CMD flask run --host 0.0.0.0 --port 8000
 Go ahead and try this example in your local Docker installation or you can use [Play with Docker](https://labs.play-with-docker.com){:target="_blank" rel="noopener" class="_"} that provides you with a temporary Docker instance on the cloud.
 
 To test this example:
+
 1. Create a file hello.py with the content of sample A.
+
 2. Create a file named Dockerfile without an extension with the contents of sample B.
+
 3. From your Docker instance build it with `docker build -t test:latest .`
```

Comment:
This is documented [here](https://kramdown.gettalong.org/syntax.html#lists). Relevant quotes:

> The content of a list item is made up of either text or block-level elements. Simple list items only contain text like in the above examples. They are not even wrapped in a paragraph tag. If the first list text is followed by one or more blank lines, it will be wrapped in a paragraph tag

> The text of the last list item is also wrapped in a paragraph tag if all other list items contain a proper paragraph as first element. This makes the following use case work like expected, i.e. all the list items are wrapped in paragraphs

Reply:
:facepalm: aha :smile:. Ideally we'd be able to explicitly mark GFM, which I don't think has some of these issues. But I can't seem to find support for this in markdown lint.

Reply:
No worries; I guess it's fine for these (mostly interested in keeping the "short" numbered lists compact).